apps/marginfi-v2-ui/src/pages/api/oracle/price.ts

```diff
@@ -102,17 +111,38 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     for (const index in requestedOraclesData) {
       const oracleData = requestedOraclesData[index];
       const priceDataRaw = oracleAis[index];
-      const oraclePrice = parsePriceInfo(oracleData.oracleSetup, priceDataRaw.data);
+      const mintData = oracleMintMap.get(oracleData.oracleKey)!;
```

Comment:
Isnt it dangerous to add ! at the end?

Reply:
it should never be undefined, feel free to correct me if I'm wrong